tests/providers/databricks/operators/test_databricks.py

```diff
@@ -2014,3 +2014,118 @@ def test_zero_execution_timeout_raises_error(self):
             "Set it instead to `None` if you desire the task to run indefinitely."
         )
         assert str(exc_info.value) == exception_message
+
+    def test_extend_workflow_notebook_packages(self):
+        databricks_workflow_task_group = MagicMock()
```

Comment:
Lets have docstrings to explain what each method tests here. Applicable for all the tests

Reply:
have added the docstrings, thanks